pymc4/random_variables/random_variable.py

```diff
@@ -157,6 +159,15 @@ def log_prob(self):
         """
         return self._transformed_distribution.log_prob(self)
 
+    def as_tensor(self):
```

Comment:
Can we leave a brief docstring here explaining why `as_tensor` is reimplemented here?

Reply:
nvm, I'm being dumb here.